lms/djangoapps/ccx/views.py

```diff
@@ -146,6 +147,9 @@ def dashboard(request, course, ccx=None):
         context['grading_policy_url'] = reverse(
             'ccx_set_grading_policy', kwargs={'course_id': ccx_locator})
 
+        with ccx_course(ccx_locator) as course:
+            context['course'] = course
```

Comment:
Thanks - I see it now.

This view uses the passed-in `course` to find the CCX - and then uses the found CCX's locator to retrieve the course again. Why not just use the passed-in course and skip the `ccx_course()` call?